src/python/gudhi/simplex_tree.pyx

```diff
@@ -409,6 +412,25 @@ cdef class SimplexTree:
         persistence_result = self.pcohptr.get_persistence()
         return self.get_ptr().compute_extended_persistence_subdiagrams(persistence_result, min_persistence)
 
+    def expansion_with_blocker(self, max_dim, blocker_func):
+        """Expands the Simplex_tree containing only a graph. Simplices corresponding to cliques in the graph are added
+        incrementally, faces before cofaces, unless the simplex has dimension larger than `max_dim` or `blocker_func`
+        returns `True` for this simplex.
+
+        The function identifies a candidate simplex whose faces are all already in the complex, inserts it with a
+        filtration value corresponding to the maximum of the filtration values of the faces, then calls `blocker_func`
+        with this new simplex (represented as a list of int). If `blocker_func` returns `True`, the simplex is removed,
+        otherwise it is kept. The algorithm then proceeds with the next candidate.
+
+        Note that you cannot update the filtration value of the simplex during the evaluation of `blocker_func`, as it
+        would segfault. 
```

Comment:
The C++ version allows that, it would be good to understand what is going wrong here.

Reply:
I guess this paragraph is now wrong?
The C++ doc has a warning about phantom simplices that should also be in the python doc.

Reply:
The note was removed on 924302f
Add a warning about phantom vertices in f6a4524